src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -695,4 +707,45 @@ private function getTemplateRegistry(string $adminCode): TemplateRegistryInterfa
 
         throw new ServiceNotFoundException($serviceId);
     }
+
+    /**
+     * @param object|array $listElement
+     *
+     * @return array array containing object and value
+     */
+    private function getObjectAndValueFromListElement(
+        $listElement,
+        FieldDescriptionInterface $fieldDescription
+    ): array {
+        if (\is_object($listElement)) {
+            $object = $listElement;
+        } elseif (\is_array($listElement)) {
+            if (!isset($listElement[0]) || !\is_object($listElement[0])) {
+                throw new \InvalidArgumentException('If listElement is an array it must contain an object on key 0');
+            }
+
+            $object = $listElement[0];
+        } else {
+            throw new \InvalidArgumentException('listElement must be an object or an array');
+        }
+
+        if (\is_array($listElement) && isset($listElement[$fieldDescription->getName()])) {
+            $value = $listElement[$fieldDescription->getName()];
+        } else {
```

Comment:
If `isset($listElement[$fieldDescription->getName()])` is `false` you shouldn't go in this `else`.

Reply:
Thanks